espresso/espresso_py.py

```diff
@@ -0,0 +1,29 @@
+from math import pi, floor
+
+class espresso:
+    cantitate_totala_cafea = 0
```

Comment:
What happens if you have multiple objects instantiated from this class?  
Will the `cantitate_totala_cafea` variable save the correct value for each object?

Reply:
added the variable as a reference to the object (self, not global anymore).